azure-pipelines.yml

```diff
@@ -20,8 +20,13 @@
 resources:
   repositories:
     - repository: templates
-      type: git   
+      type: git
       name: terracotta/terracotta
 
 jobs:
-- template: build-templates/gradle-common.yml@templates  
+- template: build-templates/gradle-common.yml@templates
+  parameters:
+    jdkVersion: '1.8'
+- template: build-templates/gradle-common.yml@templates
+  parameters:
+    jdkVersion: '11'
```

Comment:
It's 1.11, see toolchains.xml in [any build](https://dev.azure.com/TerracottaCI/terracotta/_build/results?buildId=2771)